vehicle/src/Vehicle/Backend/LossFunction/Compile.hs

```diff
@@ -217,6 +213,197 @@ compileQuant :: V.Quantifier -> Quantifier
+     compileImplication = \arg1 arg2 -> Max (Negation arg1) arg2,
+
+     compileLe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2)),
+     compileLt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2))),
+     compileGe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1)),
+     compileGt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1))),
+     compileNeq = IndicatorFunction,
+     compileEq = \arg1 arg2 -> Negation (IndicatorFunction arg1 arg2),
+
+     compileBool = \case
+                      True -> 1
+                      False -> 0
+   }
+
+lukasiewiczTranslation :: Translation
+lukasiewiczTranslation = Translation
+   {
+     compileAnd = \arg1 arg2 -> Max (Subtraction (Addition arg1 arg2) (Constant 1)) arg2,
+     compileOr  = \arg1 arg2 -> Min (Addition arg1 arg2) (Constant 1),
+     compileNot = \arg -> Subtraction (Constant 1) arg,
+     compileImplication = \arg1 arg2 -> Min (Constant 1) (Addition (Subtraction (Constant 1) arg1) arg2),
+
+
+     compileLe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2)),
+     compileLt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2))),
+     compileGe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1)),
+     compileGt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1))),
+     compileNeq = IndicatorFunction,
+     compileEq = \arg1 arg2 -> Negation (IndicatorFunction arg1 arg2),
+
+     compileBool = \case
+                      True -> 1
+                      False -> 0
+   }
+
+productTranslation :: Translation
+productTranslation = Translation
+   {
+     compileAnd = Multiplication,
+     compileOr  = \arg1 arg2 -> Subtraction (Addition arg1 arg2) (Multiplication arg1 arg2),
+     compileNot = \arg -> Subtraction (Constant 1) arg,
+     compileImplication = \arg1 arg2 -> Addition (Subtraction (Constant 1) arg1) (Multiplication arg1 arg2),
+
+     compileLe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2)),
+     compileLt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg1 arg2))),
+     compileGe = \arg1 arg2 -> Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1)),
+     compileGt = \arg1 arg2 -> Negation (Subtraction (Constant 1) (Max (Constant 0) (Subtraction arg2 arg1))),
+     compileNeq = IndicatorFunction,
+     compileEq = \arg1 arg2 -> Negation (IndicatorFunction arg1 arg2),
+
+     compileBool = \case
+                      True -> 1
+                      False -> 0
+   }
+
+yagerTranslation :: Translation
+yagerTranslation = yagerTranslationCompile 1 --change constant here
+
+--helper function that sets parameter p for the Yager DL (by default set to 1)
```

Comment:
We should have the documentation "by default set to 1" on the `yagerTranslation`, not this definition right?